docs/source/release/v3.10.0/framework.rst

```diff
@@ -32,6 +32,7 @@ Improved
 - :ref:`FilterEvents <algm-FilterEvents-v1>` now splits all the sample logs if the input splitters are given by MatrixWorkspace or a general TableWorkspace.
 - Two new properties were added to :ref:`algm-Integration` *RangeLowerList* and *RangeUpperList* can be used to give histogram-specific integration ranges.
 - :ref:`algm-FindEPP` does not output the two extra workspaces from the :ref:`algm-Fit` anymore.
+- :ref:`PredictPeaks <algm-PredictPeaks-v1>` is now faster on instruments that do not have rectangular detectors
```

Comment:
🤔 maybe if we said something along the lines of 
"The performance increase will vary instrument to instrument, Instrument X has been shown to be  Y% faster"
So people still get a relative idea of the speedup esp. with the results you saw dropping runtimes down from hours is worth putting in.